e2e-tests/functions

```diff
@@ -744,6 +744,10 @@ spinup_pxc() {
 		apply_config "$config"
 	fi
 
+	# todo(remove after 1.10 release)
+	sleep 10
+	kpatch_set_field svc "${cluster}-pxc-unready" "/spec/publishNotReadyAddresses" true
```

Comment:
It's better to detect the platform and operator version. Afterwards we will be able to make such an action